lib/b2b/otp_email.ts

```diff
@@ -0,0 +1,250 @@
+   *
+   * Currently supported languages are English (`"en"`), Spanish (`"es"`), and Brazilian Portuguese
+   * (`"pt-br"`); if no value is provided, the copy defaults to English.
+   *
+   * Request support for additional languages
+   * [here](https://docs.google.com/forms/d/e/1FAIpQLScZSpAu_m2AmLXRT3F3kap-s_mcV6UTBitYn6CdyWP0-o7YjQ/viewform?usp=sf_link")!
+   *
+   */
+  locale?: "en" | "es" | "pt-br" | string;
+}
+
+// Response type for `otps.email.loginOrSignup`.
+export interface B2BOTPEmailLoginOrSignupResponse {
+  /**
+   * Globally unique UUID that is returned with every API call. This value is important to log for debugging
+   * purposes; we may ask for this value to help identify a specific API call when helping you debug an issue.
+   */
+  request_id: string;
+  // Globally unique UUID that identifies a specific Member.
+  member_id: string;
+  /**
+   * A flag indicating `true` if a new Member object was created and `false` if the Member object already
+   * existed.
+   */
+  member_created: boolean;
+  // The [Member object](https://stytch.com/docs/b2b/api/member-object)
+  member: Member;
+  // The [Organization object](https://stytch.com/docs/b2b/api/organization-object).
+  organization: Organization;
+  /**
+   * The HTTP status code of the response. Stytch follows standard HTTP response status code patterns, e.g.
+   * 2XX values equate to success, 3XX values are redirects, 4XX are client errors, and 5XX are server errors.
+   */
+  status_code: number;
+}
+
+export class Email {
+  private fetchConfig: fetchConfig;
+  discovery: Discovery;
+
+  constructor(fetchConfig: fetchConfig) {
+    this.fetchConfig = fetchConfig;
+    this.discovery = new Discovery(this.fetchConfig);
+  }
+
+  /**
+   * Send either a login or signup email OTP to a Member. A new, pending, or invited Member will receive a
+   * signup email OTP. Non-active members will have a pending status until they successfully authenticate. An
+   * active Member will receive a login email OTP.
+   *
+   * The OTP is valid for 10 minutes. Only the most recently sent OTP is valid: when an OTP is sent, all OTPs
+   * previously sent to the same email address are invalidated, even if unused or unexpired.
+   * @param data {@link B2BOTPEmailLoginOrSignupRequest}
+   * @returns {@link B2BOTPEmailLoginOrSignupResponse}
+   * @async
+   * @throws A {@link StytchError} on a non-2xx response from the Stytch API
+   * @throws A {@link RequestError} when the Stytch API cannot be reached
+   */
+  loginOrSignup(
+    data: B2BOTPEmailLoginOrSignupRequest
+  ): Promise<B2BOTPEmailLoginOrSignupResponse> {
+    const headers: Record<string, string> = {};
+    return request<B2BOTPEmailLoginOrSignupResponse>(this.fetchConfig, {
+      method: "POST",
+      url: `/v1/b2b/otps/email/login_or_signup`,
+      headers,
+      data,
+    });
+  }
+
+  /**
+   * Authenticate a with a one-time passcode (OTP). This endpoint requires an OTP that is not expired or
```

Comment:
Looks like `{% glossary-link text="Member" termId="member" /%}` doesn't translate nicely to docstring